contributors/devel/cri-testing-policy.md

```diff
@@ -0,0 +1,108 @@
+# Container Runtime Interface: Testing Policy
+
+**Owner: SIG-Node**
+
+This document describes testing policy and process for runtimes implementing the
+[Container Runtime Interface (CRI)](/contributors/devel/container-runtime-interface.md)
+to publish test results in a federated dashboard. The objective is to provide
```

Comment:
When is the target timeline or release of this work?

Reply:
We can start doing this as soon as the proposal is approved. The majority of the change will be in the test-infra repository, which is not constrained by the k8s code freeze timeline. I think we can reasonably target this in 1.10.

At the same time, we are trying to re-categorize e2e tests (https://github.com/kubernetes/kubernetes/issues/59001) so that the node conformance suite can be more representative, and that we'll have a separate feature test suite required in the next section. The individual tests are tracked in the spreadsheet: https://docs.google.com/spreadsheets/d/1Af7zSLDEDM3i5g-8MZi0oZ7zPt85rWI1kHMQybhS9H0/edit?usp=sharing
This is also expected to be done by 1.10